packages/amplify_core/android/build.gradle

```diff
@@ -49,7 +49,7 @@ android {
 }
 
 dependencies {
-    implementation 'com.amplifyframework:core:1.1.3'
+    implementation 'com.amplifyframework:core:1.+'
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     testImplementation 'junit:junit:4.13'
     testImplementation 'org.mockito:mockito-core:3.1.0'
```

Comment:
nit: we need to eventually change the version for the testImplementation cognito version here (at the end of the dependency block)

Reply:
We need to pin the versions prior to release. I would prefer we do this in preview itself, but definitely before GA. Since we use internal APIs, there is less guarantee that things wont break.

Reply:
I think you two are referring different things. @haverchuck I've updated the `aws-auth-cognito` version to match the rest. @Ashish-Nanda  as discussed before, we will pin native library versions before the GA, not in the preview.